jenkins/Jenkinsfile

```diff
@@ -446,6 +446,12 @@ pipeline {
     
                                             # Daisy doesn't clean it up all so we remove the bucket manually
                                             gsutil rm -r $GCE_IMAGE_BUCKET
+
+                                            # Make the image public
+                                            gcloud beta compute images add-iam-policy-binding \\
```

Comment:
Why the double `\\`? Other line breaks just have 1

Reply:
Ah the IntelliJ editor seems to escape `\` when copy/pasting from GitHub. Thank you for the catch!